house_screen.py

```diff
@@ -136,13 +156,14 @@ def __init__(self):
         self.my_rock.set_current_animation(Config.RockActions.idle.value)
 
         self.brocolli = self.sprite_sheet.get_image(0,384,96,96,1,RED)
-        self.broccli_item = Item(pygame, screen, self.brocolli, 475, 175)
+        self.broccli_item = Item(pygame, screen, self.brocolli,self.my_rock, 475, 175)
         self.watering_can = self.sprite_sheet.get_image(0, 288, 96, 96, 2, RED)
-        self.watering_can_item = Item(pygame, screen, self.watering_can, 150, 600)
+        self.watering_can_item = Item(pygame, screen, self.watering_can,self.my_rock, 150, 600)
         self.ball = self.sprite_sheet.get_image(2, 288, 96, 96, 2, RED)
-        self.ball_item = Item(pygame,screen, self.ball, 300, 300)
-        self.bed = self.sprite_sheet.get_image(0,480,96,96,3,RED)
-        self.bed_item = Item(pygame,screen,self.ball,850,500)
+        self.ball_item = Item(pygame,screen, self.ball,self.my_rock,  300, 300)
+        self.bed = self.sprite_sheet.get_image(0,480,96,96,6.5,RED)
+        #self.bed = self.bed.subsurface(pygame.Rect(0,0,96,48))
+        self.bed_item = Item(pygame,screen,self.bed,self.my_rock, 875,295,False)
```

Comment:
It is nice that we caught the `self.ball` bug.

Reply:
That was a good catch.